keyboards/capsunlocked/cu80/v2_ansi/v2_ansi.c

```diff
@@ -16,3 +16,56 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 #include "v2_ansi.h"
+
+
+#ifdef CU80_RGB
+    #include "rgb_matrix_types.h"
```

Comment:
could you run a format pass on this file? (`qmk cformat (filename)` should work)

Reply:
committed as is, after running cformat